core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -263,6 +265,7 @@ public final void readFrom(StreamInput in) throws IOException {
         public static final String FROM_AS_STRING = "from_as_string";
         public static final String TO = "to";
         public static final String TO_AS_STRING = "to_as_string";
+        public static final ParseField VALUE_TYPE = new ParseField("value_type");
```

Comment:
I think this should actually be in ValuesSourceAggregatorBuilder since this `CommonFields` interface is for the response side of the aggregations but `value_type` is for the request side, so it doesn't really belong here